languages/go/internal/host/host.go

```diff
@@ -289,7 +289,7 @@ func (h Host) ToPolar(v interface{}) (*Value, error) {
 	// check composite types
 	rt := reflect.ValueOf(v)
 	// deref pointer
-	if rt.Kind() == reflect.Ptr || rt.Kind() == reflect.Interface{
+	if rt.Kind() == reflect.Ptr || rt.Kind() == reflect.Interface {
```

Comment:
did this somehow miss ci?

Reply:
We don't have a go check/lint/fmt step in CI by the looks of things.